### PR TITLE
blog: Introduction to UNS

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
As structured data in IIoT is important to make the data useful I've written a small primer on UNS.

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/blog/2023/12/introduction-to-unified-namespace.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ To facilitate a many to many connection between data producers and data consumer
 Traditionally, for example web servers serving web pages, the client requests a
 page from a server. This is a point to point connection between those two parties.
 
-<!--insert-point-to-point-image-->
+![Point to point graphic](./images/uns-point-to-point.png)
```
Comment (ZJvandeWeg) [inline src/blog/2023/12/introduction-to-unified-namespace.md:24]:
```suggestion
![Point to point graphic](./images/uns-point-to-point.png "Point to Point connection")
```

File: src/blog/2023/12/introduction-to-unified-namespace.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+authors: ["zeger-jan-van-de-weg"]
+date: 2023-12-20
+image: "/blog/2023/11/images/raspberry-pi-5-device-agent.png" #todo
+tags:
+    - posts
+    - flowfuse
+---
+
+As your organization is generating more data there’s key architectural decisions to be made to ensure the full value can be unlocked and you’re leveraging not just the tip of the iceberg. The Unified Namespace provides a blueprint to allow data to be consumed by many data-consumers.
+
```
Comment (Yndira-FlowForge) [inline src/blog/2023/12/introduction-to-unified-namespace.md:14]:
```suggestion
<!--more-->
```

File: src/blog/2023/12/introduction-to-unified-namespace.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+title: "Introduction to the Unified Namespace"
+subtitle: ""
+description: ""
```
Comment (ZJvandeWeg) [inline src/blog/2023/12/introduction-to-unified-namespace.md:4]:
```suggestion
subtitle: "Making data available for Industry 4.0 use-cases"
description: "Making data available for Industry 4.0 use-cases"
```

File: src/blog/2023/12/introduction-to-unified-namespace.md (hunk 2 above)
Comment (gdziuba) [inline src/blog/2023/12/introduction-to-unified-namespace.md]:
```suggestion

<!--more-->

```

Comment (gdziuba) [reply src/blog/2023/12/introduction-to-unified-namespace.md]: Adding for visualization

File: src/blog/2023/12/introduction-to-unified-namespace.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+date: 2023-12-20
+image: "/blog/2023/11/images/raspberry-pi-5-device-agent.png" #todo
+tags:
+    - posts
+    - flowfuse
+---
+
+As your organization is generating more data there’s key architectural decisions to be made to ensure the full value can be unlocked and you’re leveraging not just the tip of the iceberg. The Unified Namespace provides a blueprint to allow data to be consumed by many data-consumers.
+
+To facilitate a many to many connection between data producers and data consumer, there are two changes to be made for your architecture:
```
Comment (gdziuba) [inline src/blog/2023/12/introduction-to-unified-namespace.md:15]:
```suggestion
To facilitate a many to many connection between data producers and data consumers, there are two changes to be made to your architecture:
```

**Review by gdziuba — CHANGES_REQUESTED** [review]
Need the More syntax and adjusted a little bit of wording.

File: src/blog/2023/12/introduction-to-unified-namespace.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+date: 2023-12-20
+image: "/blog/2023/11/images/raspberry-pi-5-device-agent.png" #todo
+tags:
+    - posts
+    - flowfuse
+---
+
+As your organization is generating more data there’s key architectural decisions to be made to ensure the full value can be unlocked and you’re leveraging not just the tip of the iceberg. The Unified Namespace provides a blueprint to allow data to be consumed by many data-consumers.
+<!--more-->
+To facilitate a many to many connection between data producers and data consumer, there are two changes to be made for your architecture:
```
Comment (ZJvandeWeg) [inline src/blog/2023/12/introduction-to-unified-namespace.md:15]:
```suggestion
As your organization is generating more data there’s key architectural decisions to be made to ensure the full value can be unlocked and you’re leveraging not just the tip of the iceberg. The Unified Namespace provides a blueprint to allow data to be consumed by many data-consumers.

<!--more-->

To facilitate a many to many connection between data producers and data consumer, there are two changes to be made for your architecture:
```

**Review by gdziuba — APPROVED** [review]

**Pull request merged**